ProjectLighthouse.Servers.Website/Pages/Partials/UserCardPartial.cshtml

```diff
@@ -9,7 +9,7 @@
     @{
         int size = isMobile ? 50 : 100;
     }
-    <div class="cardIcon userCardIcon" style="background-image: url('/gameAssets/@Model.WebsiteAvatarHash'); min-width: @(size)px; width: @(size)px; height: @(size)px">
+    <div class="cardIcon userCardIcon" style="background-image: url('/gameAssets/@Model.WebsiteAvatarHash'); min-width: @(size)px; width: @(size)px; height: @(size)px; background-position: center center; background-size: auto @(size)px;">
```

Comment:
This should be moved to a CSS file. This is starting to get a little ridiculous, lol.